apps/sel4test-tests/src/tests/schedcontext.c

```diff
@@ -46,6 +46,14 @@ int test_sched_control_configure(env_t env)
     error = api_sched_ctrl_configure(simple_get_sched_ctrl(&env->simple, 0), sc, 5000llu, 1000llu, 0, 0);
     test_eq(error, seL4_RangeError);
 
+    /* test budget < MIN_BUDGET doesn't work */
+    error = api_sched_ctrl_configure(simple_get_sched_ctrl(&env->simple, 0), sc, 1, 5000, 0, 0);
+    test_eq(error, seL4_RangeError);
+
+    /* test budget == MIN_BUDGET does work */
+    error = api_sched_ctrl_configure(simple_get_sched_ctrl(&env->simple, 0), sc, 2 * 10, 5000, 0, 0);
```

Comment:
This should probably use a named constant/macro instead of `10`. Kernel WCET tends to be set to `10` for most platforms, but not for all of them, and it's a bit of a magic number here. @kent-mcleod is some form of the name `MIN_BUDGET` available in this scope?

Reply:
No there isn't.  Both kernel constants MIN_BUDGET_US and MAX_PERIOD_US should be available as constants in a libsel4 header somewhere but currently aren't.  Maybe for now it'd be fine to just define MIN_BUDGET_US in this file as a constant and then update it once we add the constants to libsel4?

Reply:
I added
```
#ifndef MIN_BUDGET_US
#define MIN_BUDGET_US (2 * 10)
#endif
```